src/workerd/io/trace.c++

```diff
@@ -253,9 +253,10 @@ Trace::Exception::Exception(rpc::Trace::Exception::Reader reader)
       name(kj::str(reader.getName())),
       message(kj::str(reader.getMessage())) {}
 
-Tracer::Span::Span(kj::Own<Tracer> tracer, kj::Maybe<Jaeger::SpanData> spanData)
+Tracer::Span::Span(kj::Own<Tracer> tracer, kj::Maybe<Jaeger::SpanData> spanData,
+        kj::TimePoint durationStartTime)
```

Comment:
Nit: I think we typically use 4-space indentation for line continuations, even in constructors (which looks a bit funny since it aligns with the `:`).

Reply:
Yup, I think I copy-pasted the definition from the header and neglected to tidy up the spacing.